### DISCUSSION TITLE
Error with using  GraphSAINTRandomWalkSampler

### DISCUSSION THREAD

**Discussion by freemanhm** [pyg-team/pytorch_geometric#2673] [Q&A]
Compute GraphSAINT normalization:   0%|            | 0/20956320 [00:00<?, ?it/s]
Process finished with exit code 139
This error raised when execute the code `:train_loader = GraphSAINTRandomWalkSampler(alldata, batch_size=10000, walk_length=2,
												   num_steps=100, sample_coverage=80,
												   save_dir=self.dataset.processed_dir,num_workers=0 )`
However, it would be executed successfully sometimes.

**Comment by rusty1s** [comment]
Hard to say what went wrong :( My guess is that this may be induced by isolated nodes. Does your graph contain such nodes?
